core/src/main/java/bio/terra/pearl/core/dao/dataimport/ImportDao.java

```diff
@@ -0,0 +1,22 @@
+package bio.terra.pearl.core.dao.dataimport;
+
+import bio.terra.pearl.core.dao.BaseMutableJdbiDao;
+import bio.terra.pearl.core.model.dataimport.Import;
+import org.jdbi.v3.core.Jdbi;
+import org.jdbi.v3.core.mapper.reflect.BeanMapper;
+import org.springframework.stereotype.Component;
+
+@Component
+public class ImportDao extends BaseMutableJdbiDao<Import> {
+
+    public ImportDao(Jdbi jdbi) {
+        super(jdbi);
+        jdbi.registerRowMapper(Import.class, BeanMapper.of(Import.class));
```

Comment:
(sorry I didn't catch this before) -- you don't need this line -- registering the rowMapper gets handled in the `super()`